### PR TITLE
[Jul 31] Add cloud region info

### PR DESCRIPTION
**Pull request by shainaraskas**
Adds information about available cloud regions: [cloud regions](https://elastic-dot-co-docs-production-fxkx6n78d-elastic-dev.vercel.app/current/serverless/regions)

links to cloud region docs in ES project creation docs: [link](https://elastic-dot-co-docs-production-fxkx6n78d-elastic-dev.vercel.app/current/serverless/elasticsearch/get-started#create-project)

image

Added links in obs and sec project creation docs as well: 
https://github.com/elastic/observability-docs/pull/4093
https://github.com/elastic/security-docs/pull/5604

### CODE REVIEW

**Comment by elasticdocs** [comment]
:rocket: Built _elastic-dot-co-docs-preview-docs_ successfully!

* https://elastic-dot-co-docs-production-fxkx6n78d-elastic-dev.vercel.app

* Built with commit 1da111a223266eb34f8715b587bb158baf704807

> Issues? Visit #next-docs in Slack

File: serverless/pages/cloud-regions.mdx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+slug: /serverless/regions
+title: Serverless regions
+description: Index, search, and manage ((es)) data in your preferred language.
+tags: ["serverless", "regions", "aws", "cloud"]
+---
+
+A region is the geographic area where the data center of the cloud provider that hosts your deployment is located. Review the available Elastic serverless regions to decide which region to use. If you aren't sure which region to pick, choose a one that is geographically close to you to reduce latency.
```
Comment (KOTungseth) [inline serverless/pages/cloud-regions.mdx:8]:
```suggestion
A region is the geographic area where the data center of the cloud provider that hosts your deployment is located. Review the available Elastic serverless regions to decide which region to use. If you aren't sure which region to pick, choose a one that is geographically close to you to reduce latency. 
```
```suggestion
A region is the geographic area where the data center of the cloud provider that hosts your project is located. Review the available Elastic Cloud Serverless regions to decide the region to use. If you unsure, choose a region that is geographically close to you to reduce latency. 
```

File: serverless/pages/cloud-regions.mdx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+slug: /serverless/regions
+title: Serverless regions
+description: Index, search, and manage ((es)) data in your preferred language.
+tags: ["serverless", "regions", "aws", "cloud"]
+---
+
+A region is the geographic area where the data center of the cloud provider that hosts your deployment is located. Review the available Elastic serverless regions to decide which region to use. If you aren't sure which region to pick, choose a one that is geographically close to you to reduce latency. 
+
+Elastic serverless handles all hosting details for you, and doesn't require any additional accounts with the underlying cloud provider. The region you select can't be changed after you create a project.
```
Comment (KOTungseth) [inline serverless/pages/cloud-regions.mdx:10]:
```suggestion
Elastic Cloud Serverless handles all hosting details for you, and doesn't require any additional accounts with the underlying cloud provider. You are unable to change the region after you create a project. 
```

File: serverless/pages/cloud-regions.mdx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+description: Index, search, and manage ((es)) data in your preferred language.
+tags: ["serverless", "regions", "aws", "cloud"]
+---
+
+A region is the geographic area where the data center of the cloud provider that hosts your deployment is located. Review the available Elastic serverless regions to decide which region to use. If you aren't sure which region to pick, choose a one that is geographically close to you to reduce latency. 
+
+Elastic serverless handles all hosting details for you, and doesn't require any additional accounts with the underlying cloud provider. The region you select can't be changed after you create a project. 
+
+<DocCallOut title="Note">
+Currently, only limited Amazon Web Services (AWS) regions are available. More regions for AWS, as well as Microsoft Azure and Google Cloud Platform (GCP), will be added in the future.
```
Comment (KOTungseth) [inline serverless/pages/cloud-regions.mdx:13]:
```suggestion
Currently, a limited number of Amazon Web Services (AWS) regions are available. More regions for AWS, as well as Microsoft Azure and Google Cloud Platform (GCP), will be added in the future.
```

**Review by florent-leborgne — DISMISSED** [review]
Nice addition!

File: serverless/pages/cloud-regions.mdx (hunk 1 above)
Comment (shainaraskas) [inline serverless/pages/cloud-regions.mdx:8]:
```suggestion
A region is the geographic area where the data center of the cloud provider that hosts your project is located. Review the available Elastic Cloud Serverless regions to decide which region to use. If you aren't sure which region to pick, choose a one that is geographically close to you to reduce latency. 
```

File: serverless/pages/cloud-regions.mdx [hunk 4]
```diff
@@ -0,0 +1,26 @@
+---
+slug: /serverless/regions
+title: Serverless regions
+description: Index, search, and manage ((es)) data in your preferred language.
+tags: ["serverless", "regions", "aws", "cloud"]
+---
+
+A region is the geographic area where the data center of the cloud provider that hosts your project is located. Review the available Elastic Cloud Serverless regions to decide which region to use. If you aren't sure which region to pick, choose a one that is geographically close to you to reduce latency.
```
Comment (shainaraskas) [inline serverless/pages/cloud-regions.mdx:8]:
```suggestion
A region is the geographic area where the data center of the cloud provider that hosts your project is located. Review the available Elastic Cloud Serverless regions to decide which region to use. If you aren't sure which region to pick, choose one that is geographically close to you to reduce latency. 
```

**Comment by yuvielastic** [comment]
I also see a couple of references of region in the docs; would it make sense to link the new region page to that?

- https://www.elastic.co/docs/current/serverless/observability/create-an-observability-project (Region: The region where your project will live. The region should be as close as possible to the location of your data.)
- https://www.elastic.co/docs/current/serverless/security/create-project (Region: The cloud platform’s region where your project will live. The region should be as close as possible to the location of your data. )

Also, for O11y & Security - Is there a reason why we added the following line:
> The region should be as close as possible to the location of your data.

I don't see a reason for this to be mentioned as it's customer choice to choose the region which can be far away from the location of their data. I will recommend to remove this line. 

Thoughts?

**Comment by shainaraskas** [comment]
> I also see a couple of references of region in the docs; would it make sense to link the new region page to that?
> 
> * https://www.elastic.co/docs/current/serverless/observability/create-an-observability-project (Region: The region where your project will live. The region should be as close as possible to the location of your data.)
> * https://www.elastic.co/docs/current/serverless/security/create-project (Region: The cloud platform’s region where your project will live. The region should be as close as possible to the location of your data. )
> 
> Also, for O11y & Security - Is there a reason why we added the following line:
> 
> > The region should be as close as possible to the location of your data.
> 
> I don't see a reason for this to be mentioned as it's customer choice to choose the region which can be far away from the location of their data. I will recommend to remove this line.
> 
> Thoughts?

hi @yuvielastic  - I've taken care of the link additions in these PRs, which depend on this PR to merge before they can pass CI: 

https://github.com/elastic/observability-docs/pull/4093
https://github.com/elastic/security-docs/pull/5604

I'll remove the guidance around proximity from those two pages in those PRs as well.

**Comment by KOTungseth** [comment]
> I don't see a reason for this to be mentioned as it's customer choice to choose the region which can be far away from the location of their data. I will recommend to remove this line.
> 
> Thoughts?

Yes, this is a customer choice, but we should provide guidance on the best possible region option they should choose. I think we should keep this in.

**Comment by yuvielastic** [comment]
> Yes, this is a customer choice, but we should provide guidance on the best possible region option they should choose. I think we should keep this in.
@KOTungseth But this statement isn't correct as there are multiple factors that customers can look at to identify the best possible region. 

Reasons could be:
- Data residency requirements
- Regulatory compliance
- Latency
- Cost
- Disaster recovery needs
- Proximity (location)

Some of these reasons are something that's not guided by us but chosen by customers (based on their preferences) that we are not privy to. 

So either we add these bunch of possible reasons (which would confuse customers more as they may not be necessary thinking from this guidance POV) or to keep things simple, I would suggest simply not to mention. My vote is to keep things simple and not mention the guidance at all.

**Comment by shainaraskas** [comment]
@KOTungseth + @yuvielastic in the new region page in this PR, we give them guidance around "if you don't know what to choose, consider proximity" - it's a little less prescriptive than the statements that appear in the observability and security docs. we can assume that people who need guidance will click through and read the less prescriptive recommendation there.

**Comment by KOTungseth** [comment]
> @KOTungseth + @yuvielastic in the new region page in this PR, we give them limited guidance:
> 
> > If you aren't sure which region to pick, choose one that is geographically close to you to reduce latency.
> 
> it's a little less prescriptive than the statements that appear in the observability and security docs. We can assume that people who need guidance will click through and read the less prescriptive recommendation there.

I'm good with this if @yuvielastic is good with this.

**Comment by yuvielastic** [comment]
LGTM - Thanks for improvising on it 😊

File: serverless/pages/cloud-regions.mdx [hunk 5]
```diff
@@ -0,0 +1,26 @@
+---
+slug: /serverless/regions
+title: Serverless regions
+description: Index, search, and manage ((es)) data in your preferred language.
+tags: ["serverless", "regions", "aws", "cloud"]
+---
+
+A region is the geographic area where the data center of the cloud provider that hosts your project is located. Review the available Elastic Cloud Serverless regions to decide which region to use. If you aren't sure which region to pick, choose one that is geographically close to you to reduce latency. 
+
+Elastic Cloud Serverless handles all hosting details for you, and doesn't require any additional accounts with the underlying cloud provider. You are unable to change the region after you create a project.
```
Comment (yuvielastic) [inline serverless/pages/cloud-regions.mdx:10]: I will just remove this sentence

, and doesn't require any additional accounts with the underlying cloud provider

Comment (yuvielastic) [inline serverless/pages/cloud-regions.mdx:10]: I will just remove this sentence

, and doesn't require any additional accounts with the underlying cloud provider

File: serverless/pages/cloud-regions.mdx [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+Elastic Cloud Serverless handles all hosting details for you, and doesn't require any additional accounts with the underlying cloud provider. You are unable to change the region after you create a project. 
+
+<DocCallOut title="Note">
+Currently, a limited number of Amazon Web Services (AWS) regions are available. More regions for AWS, as well as Microsoft Azure and Google Cloud Platform (GCP), will be added in the future.
+</DocCallOut>
+
+## Amazon Web Services (AWS) regions
+
+The following AWS regions are available:
```
Comment (yuvielastic) [inline serverless/pages/cloud-regions.mdx:18]: Will mention as:

The following AWS regions are currently available:

**Review by yuvielastic — APPROVED** [review]
Added 2 small comments to remove confusion. Rest LGTM

**Review by kilfoyle — APPROVED** [review]
LGTM! 🐎

**Comment by yuvielastic** [comment]
As the regions have been launched, can we please merge the documentation?

**Pull request merged**